omero/sysadmins/unix/install-web/install-fastcgi.txt

```diff
@@ -1,5 +1,12 @@
-OMERO.web FastCGI deployment
-============================
+OMERO.web FastCGI deployment (DEPRECATED)
+=========================================
+
+.. note::
```

Comment:
Do we want to start using the `deprecated` markup of Sphinx ? - see http://sphinx-doc.org/markup/para.html